src/transport/AdminPairingTable.h

```diff
@@ -190,12 +196,13 @@ class DLL_EXPORT AdminPairingInfo
         uint16_t mVendorId; /* This field is serialized in LittleEndian byte order */
 
         uint16_t mRootCertLen; /* This field is serialized in LittleEndian byte order */
-        uint16_t mOpCertLen;   /* This field is serialized in LittleEndian byte order */
+        uint16_t mICACertLen;  /* This field is serialized in LittleEndian byte order */
+        uint16_t mNOCCertLen;  /* This field is serialized in LittleEndian byte order */
 
         Crypto::P256SerializedKeypair mOperationalKey;
         uint8_t mRootCert[kMaxChipCertSize];
-        // The operationa credentials set can have up to two certs -> ICAC and NOC
-        uint8_t mOperationalCert[kMaxChipCertSize * 2];
+        uint8_t mICACert[kMaxChipCertSize];
+        uint8_t mNOCCert[kMaxChipCertSize];
```

Comment:
These should all be `kMaxChipCertSize` rather than `kMaxChipCertSize`. Please audit the other places that `kMaxChipCertSize` is used to make sure no call site expect `kMaxChipCertSize` as a constant value, when the intent was sizeof(mNOCCert/mICACert), etc.

Reply:
Updated.